src/BenchmarksClient/Startup.cs

```diff
@@ -154,9 +154,20 @@ public static int Main(string[] args)
 
                         Debug.Assert(process != null);
 
-                        process.WaitForExit();
-                        process.Dispose();
-                        process = null;
+                        try
```

Comment:
What is the overall scenario where the client job does not exit?  Is it when `wrk` hangs because it can't contact the server?  In this scenario, when will the driver set the job to `deleting`?

Reply:
If the CI stops the driver because it times out, the client job is not deleted and then block all upcoming ones.

Reply:
How does this code change fix that scenario?

Reply:
Right, that can't be it. I know when I wrote the code that the status of the job was "deleting" but it kept this way indefinitely. I assumed if the wrk process would not quit by itself it could explain this. I will try to find the CI job and the trace that had me to think this way.

Reply:
In this case I should probably also add the case when the job has been in Running state for more than the same duration. Which would be when the driver gets killed from the CI, and can't send the Delete message.

Reply:
I added this case. Please review.